frontend/ui/elements/screensaver_menu.lua

```diff
@@ -208,7 +208,7 @@ return {
                 }
             },
             {
-                text = _("Delay when exit from screensaver"),
+                text = _("Keep displaying the screensaver after wakeup"),
```

Comment:
`Keep screensaver displayed after wakeup` ?
(Somehow, this -ing feels strange for our menus, dunno if we have any :)
Fine with the others.